newsroom/default_settings.py

```diff
@@ -108,13 +108,13 @@
     'newsroom.media_utils',
 ]
 
-SITE_NAME = 'AAP Newsroom'
-COPYRIGHT_HOLDER = 'AAP'
+SITE_NAME = 'Медиасеть'
```

Comment:
you shouldn't change those default_settings, but rather add `settings.py` file to root folder and put those changes there, like in https://github.com/superdesk/newsroom/blob/ansa/settings.py

Reply:
Thanks please check now.